earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -30,6 +30,9 @@
 #include "common/SharedString.h"
 #include "StorageManagerAssetHandle.h"
 #include "AssetSerializer.h"
+#include "AssetFactory.h"
```

Comment:
I don't think this is included and we should get rid of the `using` directive below. It's generally a bad idea to have `using` directives in header files.